Smart Goods/UI/CreateRequirement/Core/CreateRequirementCore.swift

```diff
@@ -40,11 +52,16 @@ struct CreateRequirementCore: ReducerProtocol {
 
                 let requirement = getRequirement(by: scheme, state)
 
-                // TODO: do backend call
-                return EffectTask(
-                    value: .requirementSavedStateChange(.loaded(true))
-                )
-                .debounce(id: DebounceId(), for: 1, scheduler: DispatchQueue.main)
+                let call = SaveRequirementCall(httpBody: requirement)
```

Comment:
Minor: Can you create the `call` object inside the `saveRequirement` service method like in `MyRequirementService`. I know I did that but I forgot that the call itself should be in the service itself. Same goes for `checkRequirement` on line 77.